.claude/commands/handle-pr-comments.md

```
@@ -0,0 +1,138 @@
+# Handle PR Comments
+
+Handle review comments on pull requests with appropriate responses and resolutions.
+
+## Usage
+
+```
+/project:handle-pr-comments <pr-number>
+```
+
+## Overview
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)